.travis.yml

```diff
@@ -8,8 +8,11 @@ go_import_path: github.com/moolitayer/awx-client-go
 
 install:
 - go get github.com/golang/glog
+- go get golang.org/x/tools/cmd/goimports
 
 script:
+- make verify
 - go build ./awx
 - for i in examples/*.go; do go build "${i}"; done
 - go test ./awx
```

Comment:
It would be nice to pull all the script section to the make file/sh script so local users could enjoy it.

Reply:
I feel it's better to leave these as-is, so when you read the Travis output it's easier to see exactly what failed.

I modified the `Makefile` to have make targets for all of these commands, and running `make` runs them all.